### ISSUE TITLE
Clipboard item edit feature request

### ISSUE THREAD

**Issue by AndroidOL** [joecks/clipboard-actions-community#46]
1. Can I edit clipboard item in main screen?
2. On Samsung devices, DicDict is pre-install (add use local application option).
3. Bitly can't use in China, so sad.
4. Currency Converter haven't "Chinese Yuan".
5. Add Email Button and Open Link Button in main screen.
6. Add other Translator Provider, such Bing or Baidu.
7. Convert text to QR-code offline.

**Comment by AndroidOL** [comment]
Chinese translation was completed... Thanks for Developer :+1:

**Comment by joecks** [comment]
Yes now that I have chinese translations, I will also focus to add more chinese like features. Do you know what is the biggest Chinese App store? Thanks

**Comment by AndroidOL** [comment]
> I will also focus to add more chinese like features.

Thanks for your hard work!

> Do you know what is the biggest Chinese App store?

Yeah, for example:
- **Tencent** - _http://myapp.com_, named "应用宝 (Ying Yong Bao)"
-- Submit Application: I can't  

- **Wandoujia** - _http://www.wandoujia.com/apps_, named "豌豆荚 (Wan Dou Jia)"
-- Submit Application: _http://open.wandoujia.com/home_ -> _http://open.wandoujia.com/app/add_

- **Baidu** - _http://shouji.baidu.com_, named "百度手机助手 (Baidu Shou Ji Zhu Shou)"
-- Submit Application: _http://app.baidu.com/_ -> _http://app.baidu.com/apps/center/1_

- **HiAPK** - _http://apk.hiapk.com_, named "安卓市场 (An Zhuo Shi Chang / Android Shi Chang)"
-- Submit Application: _http://dev.apk.hiapk.com/login_

- **GFan** - _http://apk.gfan.com_, named "机锋市场 (Ji Feng Shi Chang)"
-- Submit Application: Need Identity Card of the PRC (China)

**Android Market in China is very confusing...**
My Recommend: _http://coolapk.com/apk/de.halfreal.clipboardactions_

BTW, Baidu.com is the largest Search Engine in China.
